### DISCUSSION TITLE
Import of TablEdit (TEF) only imports 107 of 362 bars

### DISCUSSION THREAD

**Discussion by adrian-vogler** [helge17/tuxguitar#399] [General]
I experience difficulties using the TEF-import. F.e. my latest import of TablEdit (TEF) only imports 107 of 362 bars 
[Stan Ridgway - Camouflage - StreicherG.zip](https://github.com/helge17/tuxguitar/files/15470041/Stan.Ridgway.-.Camouflage.-.StreicherG.zip)

**Comment by guiv42** [comment]
Hi
can you please create a new [issue](https://github.com/helge17/tuxguitar/issues) to track this problem?
And include in issue's description:
- which version of TuxGuitar you are using
- on which platform and which version (e.g. Windows 11, or Ubuntu 24.04, ...)
- there are 2 different .tef files in your archive, which one(s) is (are) causing the issue?

I don't have TablEdit installed, so I can't open the original song attached in your message. Could you please provide a screenshot of TablEdit showing measure 107 and the following ones? (just to see if there is something specific there which could cause the issue)

**Reply by adrian-vogler** [reply]
I will create an issue with all the details.
Both TEF files show the error.

**Comment by guiv42** [comment]
see #400
